### PR TITLE
Use Material design rules for capitalisation

### PR DESCRIPTION
**Pull request by Hixie**
See https://www.google.com/design/spec/style/writing.html#

cc @HansMuller

### CODE REVIEW

File: examples/material_gallery/lib/demo/flexible_space_demo.dart [hunk 1]
```diff
@@ -113,11 +113,11 @@ class FlexibleSpaceDemoState extends State<FlexibleSpaceDemo> {
               itemBuilder: (BuildContext context) => <PopupMenuItem<AppBarBehavior>>[
                 new PopupMenuItem<AppBarBehavior>(
                   value: AppBarBehavior.scroll,
-                  child: new Text('AppBar scrolls away')
+                  child: new Text('Toolbar scrolls away')
```
Comment (HansMuller) [inline examples/material_gallery/lib/demo/flexible_space_demo.dart]: Not sure why we're referring to the appbar as a toolbar here but OK

**Comment by HansMuller** [comment]
LGTM

File: examples/material_gallery/lib/demo/flexible_space_demo.dart (hunk 1 above)
Comment (Hixie) [inline examples/material_gallery/lib/demo/flexible_space_demo.dart]: The material spec uses the term "Toolbar", I was just making the demo match the spec. Happy to change it back if you want.

Comment (HansMuller) [inline examples/material_gallery/lib/demo/flexible_space_demo.dart]: I have a feeling this doesn't matter, so don't change it. The "app bar" term, along with some diagrams that imply that app bar's contain toolbars, can be found here: https://www.google.com/design/spec/patterns/scrolling-techniques.html

It's not all perfectly consistent but there is this:
> The standard app bar is the overarching component that can include the following blocks: a toolbar, tab bar, or flexible space.

Comment (Hixie) [inline examples/material_gallery/lib/demo/flexible_space_demo.dart]: Oh, interesting. Ok. My bad. I'll file a bug on myself to fix it.

Comment (Hixie) [inline examples/material_gallery/lib/demo/flexible_space_demo.dart]: https://github.com/flutter/flutter/issues/3161

**Pull request merged**